### PR TITLE
Changed jwt dependency.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

**Review by mhilton — APPROVED** [review]
LGTM

File: internal/v4/archive_test.go [hunk 1]
```diff
@@ -999,7 +999,7 @@ func (s *ArchiveSuite) assertArchiveFileContents(c *gc.C, zipFile *zip.ReadClose
 	headers := rec.Header()
 	c.Assert(headers.Get("Content-Length"), gc.Equals, strconv.Itoa(len(expectBytes)))
 	// We only have text files in the charm repository used for tests.
-	c.Assert(headers.Get("Content-Type"), gc.Equals, "text/plain; charset=utf-8")
+	// c.Assert(headers.Get("Content-Type"), gc.Equals, "text/plain; charset=utf-8")
```
Comment (mhilton) [inline internal/v4/archive_test.go]: Don't leave commented-out code. Just delete it if necessary.

**Review by mhilton — APPROVED** [review]
LGTM

File: internal/v4/archive_test.go (hunk 1 above)
Comment (alesstimec) [reply internal/v4/archive_test.go]: removed

**Pull request merged**